backend/src/main/java/pl/zzpj/esportbetting/response/MatchResponse.java

```diff
@@ -0,0 +1,25 @@
+package pl.zzpj.esportbetting.response;
+
+import lombok.Builder;
+import lombok.Data;
+import pl.zzpj.esportbetting.model.Comment;
+import pl.zzpj.esportbetting.model.Team;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder
+public class MatchResponse {
+    private long id;
+    private LocalDateTime startDate;
+    private int realId;
+    private boolean isFinished;
+    private int realScoreA;
+    private int realScoreB;
+    private Team teamA;
+    private Team teamB;
+    private int stakeA;
+    private int stakeB;
+    private List<Comment> comments;
+}
```

Comment:
Takie pytanko.
Czy naprawdę potrzebna nam ta klasa?
Zawiera ona dokładnie takie same pola jak klasa ```Match```, czemu więc w RestController nie przesłać obiektu właśnie typu ```Match```?
Nie krytykuję, tylko pytam. Może jest coś, czego nie wiem.

Reply:
Masz rację, nie potrzebne to. Usunięte